### ISSUE TITLE
Move babel config to package.json

### ISSUE THREAD

**Issue by SaFrMo** [funkhaus/vuepress#92]
@johndigital Wanted to check before just doing so - any reason you can think of not to keep the babel config in `package.json`? https://babeljs.io/docs/usage/babelrc/

**Comment by johndigital** [comment]
Nope, I'm okay with moving it to package.json! Will help keep our root directory clean.
